client/my-sites/controller.js

```diff
@@ -269,7 +270,8 @@ module.exports = {
 		const siteFragment = context.params.site || route.getSiteFragment( context.path );
 		const basePath = route.sectionify( context.path );
 		const currentUser = user.get();
-		const hasOneSite = currentUser.visible_site_count === 1;
+		const visibleSiteCount = getVisibleSites( getState() ).length;
```

Comment:
Optional, but it looks like we call getState() multiple times. We can store this in a local var:
```
const state = getState();
```